e-moe/guzzle6-bundle/1.1/etc/packages/emoe_guzzle_bundle.yaml

```diff
@@ -0,0 +1,4 @@
+emoe_guzzle:
+    log:
+        enabled: true # Logging requests to Monolog
+        format: 'Guzzle: [{ts}] "{method} {uri} HTTP/{version}" {code}' # Optional log format customization
```

Comment:
@javiereguiluz but then, it forces all developers joining a project to wonder "why is this config commented in our config file ?".
So I'm more in favor of removing the config entirely. People wanting to customize the log format would then read the doc (to know that it is possible, but also to know the available placeholder, which still requires reading the doc anyway)